docs/editors/emacs.md

```diff
@@ -0,0 +1,124 @@
+---
+id: emacs
+title: Emacs
+---
+
+Metals works in Emacs thanks to the the
+[`lsp-scala`](https://github.com/rossabaker/lsp-scala) package.
+
+![Emacs demo](../assets/emacs-demo.gif)
+
+> The Emacs LSP client has [several known issues](#known-issues). Most
+> critically, diagnostics are not published for unopened buffers meaning compile
+> errors can get lost.
+
+```scala mdoc:requirements
+
+```
+
+## Installation
+
+First, clone the `lsp-scala` repository.
+
+```sh
+git clone https://github.com/rossabaker/lsp-scala.git
+```
+
+Next, update your Emacs configuration to load `lsp-scala` along with its
+dependencies
+
+```el
```

Comment:
I dumped everything I ended up needing in a clean `init.el` to all the functionality working. It can definitely be improved, and ideally simplified once `lsp-scala` is up on MELPA.

I have no experience with emacs so I suspect this hacky config would be illegal is certain Emacs circles.

Any thoughts @rossabaker?